context/app/routes_notebooks.py

```diff
@@ -50,12 +52,35 @@ def notebook(entity_type):
     search_url = (
         current_app.config['ELASTICSEARCH_ENDPOINT']
         + current_app.config['PORTAL_INDEX_PATH'])
+
+    # Common to all entities:
     cells = [
         new_markdown_cell(
             f'This notebook demonstrates how to work with HuBMAP APIs for {entity_type}:'),
         new_code_cell('!pip install requests'),
         *get_shared_cells(uuids=uuids, url_base=url_base, entity_type=entity_type),
     ]
+
+    # Only for datasets:
     if entity_type == 'datasets':
         cells += get_file_cells(search_url=search_url)
+
+    # Only for datasets with anndata:
+    uuids_to_zarr_files = _get_uuids_to_zarr_files(uuids)
+    zarr_files = set().union(*uuids_to_zarr_files.values())
+    if zarr_files:
+        cells += get_anndata_cells(uuids_to_zarr_files)
+
     return _nb_response(entity_type, cells)
+
+
+def _get_uuids_to_zarr_files(uuids):
```

Comment:
doesn't this actually return the full oath to the zarr files? so in this sense, we are not returning uuids but paths?  maybe i'm misreading the function name or something